features/config/developer_can_use_composer_to_create_suites.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Composer can be leveraged to create suites
+  As a Developer
+  I need the autoload rules I defined to be reused by phpspec
+  So I may enable the Composer namespace provider and get one suite per autoload rule
+
+  Scenario: Using composer as namespace_provider
+    Given the config file located in "." contains:
+      """
+      namespace_providers:
+          - composer: ~
+      """
+    When I start describing the "MilkyWay/OrionCygnusArm/LocalBubble" class with the "phpspec.yml" custom config
+    Then a new spec should be generated in the "spec/LocalBubbleSpec.php":
```

Comment:
I just fixed this expectation, that was plain wrong, and now see that I must compute the psr4 prefix by analyzing the location and the namespace.